<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter 1st number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btnAdd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSub">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btnSub</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMul">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btnMul</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDiv">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btnDiv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btnClear</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter 2nd Number </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
